content/docs/pulumi-cloud/get-started/onboarding-guide.md

```diff
@@ -0,0 +1,326 @@
+
+Pulumi Cloud makes it easy to collaborate on your cloud projects faster and with more confidence. Typically the infrastructure platform group will own the configuration of Pulumi and onboarding developers and security experts to facilitate building secure cloud applications and infrastructure. The onboarding process begins with configuring your organization.
+
+### Organizing people for collaboration
+
+Your Pulumi Cloud hierarchy is made up of individual **users**, **organizations**, and **teams**. Each plays a critical role in how your team collaborates and enforcing security and best practices.
+
+You will begin by creating an organization. An organization is first and foremost a collection of individual user members. It is a container for IaC projects and stacks, ESC projects and environments, and Insights accounts and discovered resources. Most companies only need a single organization, although it is possible to have multiple if you need extra separation or multiple identity providers. Learn more about [organizations here](https://www.pulumi.com/docs/pulumi-cloud/admin/organizations/).
+
+An organization can also contain teams. A team is a group of individual users that typically reflects your company’s structure or working groups. By grouping members into teams, you can control which projects, stacks, and environments they have access to, and with what permissions, through role-based-access-control (RBAC). We know that teams and roles change frequently, so Pulumi Cloud makes it easy to reconfigure teams as needed, or even have them automatically sync with your identity provider of choice. Learn more about [teams here](https://www.pulumi.com/docs/pulumi-cloud/access-management/teams/).
+
+Start by creating [your organization here](https://app.pulumi.com/?create-organization=1). Organizations and their capabilities are a paid feature of Pulumi Cloud, so you will notice that creating an organization starts a free trial. Pulumi has a generous free tier when the trial expires and you can always [contact us](https://pulumi.com/contact) if you need more time.
+
+### Configuring Single Sign-On (SSO)
+
+Many organizations prefer to use Single Sign-On (SSO) to leverage an identity provider such as Azure Active Directory, Google Workspace, Okta, or any other SAML 2.0 compliant provider. To learn about configuring SSO for your organization, [read Pulumi Cloud SAML(SSO)](https://www.pulumi.com/docs/pulumi-cloud/access-management/saml/). If you’d like to continue using email, GitHub, GitLab, or Atlassian identity, you can skip this step.
+
+### Inviting your teammates
+
+As soon as you’ve completed these set up steps, you are ready to invite your team\! If you are using SSO with System for Cross-domain Identity Management (SCIM) enabled, onboarding and offboarding happen automatically. Otherwise, refer to [Inviting members to an organization](https://www.pulumi.com/docs/pulumi-cloud/admin/organizations/#inviting-members-to-an-organization).
```

Comment:
```suggestion
As soon as you’ve completed these set up steps, you are ready to invite your team\! If you have SSO with System for Cross-domain Identity Management (SCIM) enabled, onboarding and offboarding happen automatically. Otherwise, refer to [Inviting members to an organization](https://www.pulumi.com/docs/pulumi-cloud/admin/organizations/#inviting-members-to-an-organization).
```